ignite/metrics/metric.py

```diff
@@ -169,7 +169,7 @@ def __init__(self, *args, **kwargs):
                     super().__init__(*args, **kwargs)
 
                 def update(self, output):
-                    y_pred, y, x = output
+                    y_pred, y, x = output["y_pred"], output["y"], output["x"]
```

Comment:
Seems like this example is a bit confusing. It was correct as `update` function receives a tuple.